pkgs/by-name/tr/trunk-ng/package.nix

```diff
@@ -20,7 +20,7 @@ rustPlatform.buildRustPackage rec {
   # requires network
   checkFlags = [ "--skip=tools::tests::download_and_install_binaries" ];
 
-  cargoHash = "sha256-37nCqRTgbsg2cXu4xwYC/qfodPIxx97Qns8FQe9NroQ=";
+  cargoHash = "sha256-O3e8v0r76VeMYODah2RYTmwr9WNAX+HPhYVmDuP2gfg=";
```

Comment:
I believe he wants you to change the cargo hash to what he's suggesting.

Reply:
It's an automatic PR, so I am not sure I can make changes to it.

Reply:
I wasn't able to push changes to this PR, so I created a new one :shrug: